docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,25 @@
 Version history
 ===============
 
+5.3.0-m2 (2016 November 23)
```

Comment:
I assumed we'd amalgamate the list for the 5.3.0 release but I can do it as we go along if you prefer.